scripts/SANS/ISISCommandInterface.py

```diff
@@ -218,6 +218,22 @@ def SetFrontDetRescaleShift(scale=1.0, shift=0.0, fitScale=False, fitShift=False
     _printMessage('#Set front detector rescale/shift values')
 
 
+def SetMergeQRange(q_max=None, q_min=None):
+    """
+        Stores property about the detector which is used to rescale and shift
+        data in the bank after data have been reduced
+        @param scale: Default to 1.0. Value to multiply data with
+        @param shift: Default to 0.0. Value to add to data
+        @param fitScale: Default is False. Whether or not to try and fit this param
+        @param fitShift: Default is False. Whether or not to try and fit this param
+        @param qMin: When set to None (default) then for fitting use the overlapping q region of front and rear detectors
+        @param qMax: When set to None (default) then for fitting use the overlapping q region of front and rear detectors
```

Comment:
What are these params? I only see `q_max` and `q_min` in the signature above.

Also: I think it is also customary to order the min and max arguments as `def SetMergeQRange(q_min, q_max)`. Is there a specific reason why they are the other way here?